jerry-core/ecma/operations/ecma-bigint.c

```diff
@@ -147,6 +147,15 @@ ecma_bigint_parse_string (const lit_utf8_byte_t *string_p, /**< string represena
         digit = (ecma_bigint_digit_t) (character - (LIT_CHAR_LOWERCASE_A - 10));
       }
```

Comment:
I would add the code here with an else:
```
else
{
  const lit_utf8_byte_t *possible_end = ecma_string_trim_front (string_p, string_end_p);
  if (possible_end == string_end_p) ...
}
```
Note: there is no need to check the next character is a white space because `ecma_string_trim_front` will do it automatically

Reply:
Just to think of it, you don't even need `possible_end` variable, just compare `ecma_string_trim_front (string_p, string_end_p) == string_end_p`